openlitespeed/README.md

```diff
@@ -0,0 +1,45 @@
+OpenLiteSpeed
+===============
+[OpenLiteSpeed](https://github.com/litespeedtech/openlitespeed) is the Open Source edition of [LiteSpeed Web Server Enterprise](https://www.litespeedtech.com/). 
+More information about OpenLiteSpeed can be found [here](https://openlitespeed.org/).
+
+## Purpose 
+This directory contains a Dockerfile that builds [OpenLiteSpeed](https://github.com/litespeedtech/openlitespeed) with [OQS-BoringSSL](https://github.com/open-quantum-safe/boringssl), which allows OpenLiteSpeed to negotiate quantum-safe key exchange using [liboqs](https://github.com/open-quantum-safe/liboqs/).
+
+
+
+## Getting started
+
+## Server
+### Building
+Assuming Docker is [installed](https://docs.docker.com/install) the following command
+
+```
+docker build -t lsws -f Dockerfile-server .
+docker network create lsws-test
+docker run --network lsws-test --name lsws -it lsws bash
+```
+
+will run the container for the quantum-safe crypto (QSC) protected OpenLiteSpeed server on the docker network called lsws-test.
+
+### Usage
+Documentation for using the server docker image is contained in the separate [USAGE-server.md](https://github.com/open-quantum-safe/oqs-demos/blob/main/openlitespeed/USAGE-server.md) file.
```

Comment:
This is a hard coded absolute link: It should be possible to make this relative (so folder name can change in the future and the link also works in your own repo).

Reply:
Noted. I made the requested changes.